Providers/Profile.swift

```diff
@@ -974,20 +1051,25 @@ open class BrowserProfile: Profile {
             }
             return d
         }
-
+        
         fileprivate func syncLoginsWithDelegate(_ delegate: SyncDelegate, prefs: Prefs, ready: Ready, why: SyncReason) -> SyncResult {
             log.debug("Syncing logins to storage.")
             return syncUnlockInfo().bind({ result in
                 guard let syncUnlockInfo = result.successValue else {
                     return deferMaybe(SyncStatus.notStarted(.unknown))
                 }
 
-                return self.profile.logins.sync(unlockInfo: syncUnlockInfo).bind({ result in
+                return self.profile.logins.sync(unlockInfo: syncUnlockInfo).bind({ [weak self] result in
                     guard result.isSuccess else {
                         return deferMaybe(SyncStatus.notStarted(.unknown))
                     }
 
                     let syncEngineStatsSession = SyncEngineStatsSession(collection: "logins")
+                    if #available(iOS 12, *), let self = self {
+                        self.profile.syncCredentialIdentities().upon { result in
+                            print(result)
```

Comment:
What are we printing here? Could we use the logger to only print in debug?

Reply:
Now using `log.debug()` instead.